app/middleware/ackama/http_basic_auth.rb

```diff
@@ -0,0 +1,27 @@
+module Ackama
```

Comment:
I'm unsure about the benefit of the namespace here vs just `HttpBasicAuth`. I think that in our work for clients we should probably not be wacking our own company namespaces across things?

Reply:
My thinking was that we should call out the "from a template" nature of this for devs who use it but upon reflection I think you are right. Will change.